submissions/Roophee/friends-app/index.js

```diff
@@ -1,15 +1,23 @@
 'use strict';
 
 const DATA_URL = 'https://randomuser.me/api/?results=100';
-const RAW_DATA = [];
-const USER_STOR = new Map();
+let RAW_DATA = [];
+const NODES_STORAGE = new Map();
 const mainSection = document.querySelector('.main__section');
-const filterSettings = {searchParam: '',
-                        genderParam: '',
-                        ageParam: '',
-                        nameParam: '',
-                        filteredData: [],
-                        secondFilteredData: []};
+const filterSettings = {
+searchParam: '',
+genderParam: '',
+ageParam: '',
+nameParam: '',
+filteredData: [],
+secondFilteredData: [],
+filterOne: '',
+filterTwo: '',
+};
+const wasFirstFiltration = () => filterSettings.filteredData.length >= 1;
+const wasSecondFiltration = () => filterSettings.secondFilteredData.length >= 1;
```

Comment:
1. verb
1. `wasFiltration` ? maybe `getIsFiltered` will be better name for this one
1. if you use functions you can do things like this
```suggestion
const getIsFiltered = (data) => data.length >= 1;
```
or 
```suggestion
const getIsFiltered = (settingDataKey) => filterSettings[settingDataKey].length >= 1;
```